ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestContext.scala

```diff
@@ -69,7 +69,7 @@ object LedgerTestContext {
 
 }
 
-private[infrastructure] final class LedgerTestContext(
+private[testtool] final class LedgerTestContext(
```

Comment:
Not sure this change is necessary. If we can avoid allowing test contexts to be instantiated outside of the `infrastructure` package I believe it would make it less probable to inadvertently shoot ourselfs in the foot.

Reply:
I only needed that to be able to use the type `LedgerTestContext` in the signature of a [helper method](https://github.com/digital-asset/daml/pull/2580/files#diff-1b8839de3a7ecbb5a49106dd66e0dc7cR52). How about making the constructor package private to `infrastructure`?

```scala
private[testtool] final class LedgerTestContext private[infrastructure](...)
```

Reply:
Makes sense, thanks.